sdk/keyvault/azure-security-keyvault-secrets/test/ut/CMakeLists.txt

```diff
@@ -18,16 +18,8 @@ add_executable (
   azure-security-keyvault-secrets-test
     challenge_based_authentication_policy_test.cpp
     macro_guard.cpp
-    secret_backup_deserialize_test.cpp
-    secret_backup_deserialize_test.hpp
     secret_client_base_test.hpp
     secret_client_test.cpp
-    secret_get_client_deserialize_test.cpp
-    secret_get_client_deserialize_test.hpp
-    secret_paged_deserialize_test.cpp
```

Comment:
What is the plan for these deleted tests?

Reply:
those tests became irelevant, previously the serializer were in separate classes to make the code more readable, and avoid duplication. Now the deserialization is generated and is part of the methods, thus they are not separate methods that exist independent of the API surface. 

Thus i had to remove them . The code coverage went up actually proving that that code is exercised as part of the client tests